src/main/webapp/WEB-INF/register.jsp

```diff
@@ -9,27 +9,65 @@
 </head>
 <body>
 
-<%  String username2="";
-    String first_name="";
-    String last_name="";
-    String email="";
-    String street_address="";
-    String zip_code="";
-    String state="";
-    String phone="";
-    String password="";
-    String confirm_password="";
-
-    username2 = request.getParameter("username");
-    first_name = request.getParameter("first_name");
-    last_name = request.getParameter("last_name");
-    email = request.getParameter("email");
-    street_address = request.getParameter("street_address");
-    zip_code = request.getParameter("zip_code");
-    state = request.getParameter("state");
-    phone = request.getParameter("phone");
-    password = request.getParameter("password");
-    confirm_password = request.getParameter("confirm_password");
+<%  String username_temp=request.getParameter("username");
```

Comment:
This code (lines 12 to 70) should be in your Servlet. Please move it to the Servlet and test your changes again.